README.md

```diff
@@ -15,37 +15,19 @@
     ·
     <a href="https://github.com/TheNerdful8/Lecture2Gether/pulls">Send a Pull Request</a>
   </p>
+  <p align="center">
+    <img src="static/L2Ge_frontpage.png" alt="Screenshot Frontpage" height="200">
+    <img src="static/L2Ge_player.png" alt="Screenshot Player" height="200">
+  </p>
 </p>
 
 ## About The Project
-<!-- TODO add screenshot -->
 
-Lecture2Gether makes it possible to watch online lectures with friends by pasting a link to a [Lecture2Go](https://github.com/lecture2go/portal-6.2-ce-ga6) Video, a YouTube Video or a simple mp4 link.
+Lecture2Gether makes it possible to watch online lectures with friends by pasting a link to a [Lecture2Go](https://github.com/lecture2go/portal-6.2-ce-ga6) Video, a YouTube or Google Drive Video or a simple .mp4 link.
 The video streams are synchronized to partially restore the social aspect of campus life.
 
 [![Flask CI](https://github.com/TheNerdful8/Lecture2Gether/workflows/Flask%20CI/badge.svg?branch=master)](https://github.com/TheNerdful8/Lecture2Gether/actions?query=workflow%3A%22Flask+CI%22) &nbsp;&nbsp; [![Node.js CI](https://github.com/TheNerdful8/Lecture2Gether/workflows/Node.js%20CI/badge.svg)](https://github.com/TheNerdful8/Lecture2Gether/actions?query=workflow%3A%22Node.js+CI%22) &nbsp;&nbsp; [![Kubernetes configuration](https://github.com/TheNerdful8/Lecture2Gether/workflows/Kubernetes%20configuration/badge.svg)](https://github.com/TheNerdful8/Lecture2Gether/actions?query=workflow%3A%22Kubernetes+configuration%22)
```

Comment:
Maybe we should move the badges to the top. They look weird here and I think most projects have them further up.

Reply:
We experimented with this a bit in the PR that added the badges, but they kind of broke the clean centered "logo only" style of the top part. Maybe they look better now since more badges got added. Also, the badges cloud be centered under the "submit a bug" part. or something like this.